packages/near-membrane-base/src/membrane.ts

```diff
@@ -152,6 +158,8 @@ export function init(
     const proxyTargetToPointerMap = new WeakMap();
     const LockerLiveValueMarkerSymbol = Symbol.for('@@lockerLiveValue');
     const distortionCallback = optionalDistortionCallback || ((o) => o);
+    const InboundInstrumentation = `to:${color}`;
```

Comment:
@garychangsf @dbajric this is what we have discussed offline, instrumenting from blue side, but instrumenting inbound and outbound operations from blue rather than blue-to-red and red-to-blue. In theory this are equivalent anyways.